apps/krew-website/src/pages/index.astro

```diff
@@ -1,7 +1,7 @@
 ---
-import Head from '$components/Head.astro';
-import Header from '$components/Header.astro';
-import Footer from '$components/Footer.astro';
+import Head from '../components/Head.astro';
```

Comment:
haven't figured out how to do aliases in our nx monorep yet (because I don't want to add some 'generic' path names to the root tsconfig file.